### ISSUE TITLE
dotnet core compatibility 

### ISSUE THREAD

**Issue by merl111** [sbarisic/NuklearDotNet#10]
Hi, any plans on moving to dotnet core or to dual target with?

**Comment by sbarisic** [comment]
I do not plan on adding .net core support myself, but i would be accepting pull requests

**Comment by merl111** [comment]
Fair enough, PR follows.

**Comment by wave9d** [comment]
Any update on this  ?

**Comment by sbarisic** [comment]
> Any update on this ?

No, not yet, quite busy with work and other stuff right now. But i am accepting pull requests if somebody would be willing to invest time in this.
